nibabel/parrec.py

```diff
@@ -609,7 +609,7 @@ def __getitem__(self, slicer):
                                  'F')
         # Broadcast scaling to shape of original data
         slopes, inters = self._slice_scaling
-        fake_data = strided_scalar(self._shape)
```

Comment:
Yeah. Does it make sense to try to find another low-memory option, here, or is this something that needs to be addressed in numpy? And in the meantime, should we skip this test on `numpy >= 1.10`, use `np.zeros`, or put up with failing builds?